### ISSUE TITLE
Unknown profiles show up in destructiveChanges.xml as ApexClass when using delta-push

### ISSUE THREAD

**Issue by pcon** [solenopsis/Solenopsis#162]
When using `delta-push` and a profile does not exist in the target environment the profile shows up twice in the `destructiveChanges.xml` file

Example `destructiveChanges.xml`

```
<package xmlns="http://soap.sforce.com/2006/04/metadata">
    <types>
        <members>Post Chat Window Profile1359720664216</members>

        <name>ApexClass</name>
    </types>
    <types>
        <members>Post Chat Window Profile1359720664216</members>

        <name>Profile</name>
    </types>
    <version>30.0</version>
</package>
```

**Comment by sfloess** [comment]
I believe this will be solved once Issue #163 is fixed.  We can wait and close later :+1:

**Comment by sfloess** [comment]
I am fairly certain my changes for Issue #163 will resolve this.  As an example, I removed Integration.profile and SolutionManager.profile (as well as a few other files in testing).  Below is my destructiveChanges.xml:

```xml
<package xmlns="http://soap.sforce.com/2006/04/metadata">
    <types>
        <members>Integration</members>
        <members>SolutionManager</members>
        <name>Profile</name>
    </types>
    <types>
        <members>NegotiatedEntitlementProcess</members>
        <members>Task</members>
        <name>ApexTrigger</name>
    </types>
    <types>
        <members>Daily_Breaches__c</members>
        <name>Workflow</name>
    </types>
    <version>33.0</version>
</package>
```
